pygmt/helpers/decorators.py

```diff
@@ -575,6 +575,17 @@ def new_module(*args, **kwargs):
                         f"Use long-form parameter '{long_alias}' instead."
                     )
                     warnings.warn(msg, category=SyntaxWarning, stacklevel=2)
+
+            # timestamp (U) is deprecated since v0.8.0.
```

Comment:
Indicate the deprecation minor version, assuming we're gonna do 4 minor versions?
```suggestion
            # timestamp (U) is deprecated since v0.8.0, remove in v0.12.0
```